pkg/keycloak/phaseHandler.go

```diff
@@ -89,7 +89,11 @@ func (ph *phaseHandler) Accepted(sso *v1alpha1.Keycloak) (*v1alpha1.Keycloak, er
 	}
 
 	kc.Spec.AdminCredentials = adminCredential.GetName()
-	kc.Status.Phase = v1alpha1.PhaseAwaitProvision
+	var nextPhase = v1alpha1.PhaseAwaitProvision
```

Comment:
might as well just assign this straight into the `kc.status.Phase` rather than using the intermediary `nextPhase` var.